extensions/wikia/PortableInfobox/tests/PortableInfoboxDataServiceTest.php

```diff
@@ -33,7 +33,9 @@ public function testEmptyData() {
 	}
 
 	public function testLoadFromProps() {
-		$data = '[{"data": [], "sources": []}]';
+		$data = '[{"parser_tag_version": ' .
+			PortableInfoboxParserTagController::PARSER_TAG_VERSION .
+			', "data": [], "metadata": []}]';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal ", "data": [], "metadata": []}]" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)